manager/src/main/java/com/redhat/service/bridge/manager/actions/connectors/SlackAction.java

```diff
@@ -54,6 +56,8 @@ public JsonNode connectorPayload(BaseAction action) {
         objectNode.set(CONNECTOR_WEBHOOK_URL_PARAMETER, new TextNode(webHookURL));
         objectNode.set(CONNECTOR_TOPIC_PARAMETER, new TextNode(kafkaTopic));
 
+        addLogProcessorToDefinition(mapper, objectNode);
```

Comment:
Is it possible to centralise this at `ConnectorAction` level? To avoid to configure it explicitly in each action based on Connector. 
Maybe something like a default `postConnectorPayload` or similar